app/styles/partials/footer.scss

```diff
@@ -27,6 +27,10 @@ footer {
   }
 }
 
+.ui.inverted.link.list {
+  text-align: left;
+}
+
```

Comment:
I think we can fix it with Semantic UI classes.

Reply:
Should I  add a new class to the columns and then add styling to that class, is that what you mean ? I am a beginner and thought to solve this as my first issue

Reply:
We use `Semantic UI` which is like `bootstrap`. So just like bootstrap classes, you can use semantic UI classes. More info https://semantic-ui.com/collections/grid.html

Reply:
oh thanks I got that , just working on it only

Reply:
> We use `Semantic UI` which is like `bootstrap`. So just like bootstrap classes, you can use semantic UI classes. More info https://semantic-ui.com/collections/grid.html

Just commited changes using Semantic UI classes as suggested by you , please review .